TriblerGUI/widgets/trustpage/js/radial_view/radial_link.js

```diff
@@ -0,0 +1,238 @@
+if(typeof require !== "undefined") {
+    var config = require("../style_config.js");
+    var applyEventfulMixin = require("../support/eventful_mixin.js").applyEventfulMixin;
+}
+
+/**
+ * The links between nodes in the Radial View.
+ * @param svg
+ * @constructor
+ */
+function RadialLinks(svg, options) {
+
+    applyEventfulMixin(this);
+
+    var self = this;
+
+    var defaults = {};
+
+    self.graphData = null;
+
+    self.config = Object.assign({}, defaults, options || {});
+
+    /**
+     * Select all <svg.link> objects in the SVG
+     */
+    self.selectAll = function () {
+        return svg.select(".links").selectAll(".link");
+    };
+
+    /**
+     * Create, update and destroy the <svg.link> objects based on new data
+     * @param {GraphData} newGraphData
+     */
+    self.onNewData = function (newGraphData) {
+        self.graphData = newGraphData;
+
+        var update = self.selectAll()
+            .data(self.graphData.links, function (link) {
+                return link.id;
+            });
+
+        self.destroy(self.selectAll());
+        self.update(update);
+        self.create(update.enter());
+    };
+
+    /**
+     * Create the new <svg.link> objects based on entering data
+     * @param enterSelection
+     */
+    self.create = function (enterSelection) {
+        // Create <svg.link>
+        var links = enterSelection
+            .append("svg")
+            .attr("class", "link")
+            .style("opacity", "0")
```

Comment:
nope, on line 84 it is after a transition. So line 56 says: it starts at 0. Line 84 says: after 1000 ms set it to a different value.